app/src/main/java/com/example/greehousecontroller/MainActivity.java

```diff
@@ -66,13 +76,19 @@ public void onStart() {
         }
         else{
             navController.navigate(R.id.nav_login);
+            toolbar.setVisibility(View.GONE);
```

Comment:
This just hides it from view, but you can still go back using the phones back button